### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

**Comment by Psycojoker** [comment]
Thanks for your work :)

Don't hesitate to modify the .travis-ci.yml to include other version of python in the CI build.

**Comment by rowanseymour** [comment]
I've added tox to test against Python 2.7 and 3.4. 

Two of the tests for 3.4 are failing because of https://github.com/Psycojoker/django-hamlpy/issues/2

**Comment by rowanseymour** [comment]
@Psycojoker I've merged in #3 so tests pass now. Ready for review.

**Comment by Psycojoker** [comment]
Great, will try to do that pretty fast :) (I'm at work right now and this isn't our current focus)

File: hamlpy/elements.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
+from __future__ import print_function, unicode_literals
+
 import re
+import six
```
Comment (Psycojoker) [inline hamlpy/elements.py]: Six seems to be missing from `setup.py` and from the `requirements` file.

This apparently come from the fact that `mock` pulls it for testing.

**Comment by rowanseymour** [comment]
I've added six as a requirement. I've got a few suggested changes to the readme.md - can I put those in a subsequent PR ?

**Comment by Psycojoker** [comment]
> I've got a few suggested changes to the readme.md - can I put those in a subsequent PR ?

Yup :)

Do you think that this code is ready for release of would you prefer to wait a bit before?

Thanks again for your work!

**Comment by rowanseymour** [comment]
I think it is. The only way we're going to find Python 3 issues is to start using it! In fact I'd appreciate a PyPI release before we merge the attribute parser stuff because I'm less confident about it - it's a lot of changes.

I'd suggest bumping the minor version so this is the 0.9.0 release.

**Comment by Psycojoker** [comment]
>  I'd appreciate a PyPI release before we merge the attribute parser stuff

And that's done :) https://pypi.python.org/pypi/django-hamlpy/0.85

> I'd suggest bumping the minor version so this is the 0.9.0 release.

Actually, we are at number "85" and not "8", jesse seems to have done a lot of releases.

**Comment by rowanseymour** [comment]
Oh yeah we are. Weird. Thanks for the release!

**Pull request merged**